led_strip/examples/led_strip_rmt_ws2812/main/led_strip_rmt_ws2812_main.c

```diff
@@ -23,11 +23,12 @@ led_strip_handle_t configure_led(void)
 {
     // LED strip general initialization, according to your led board design
     led_strip_config_t strip_config = {
-        .strip_gpio_num = LED_STRIP_BLINK_GPIO,   // The GPIO that connected to the LED strip's data line
-        .max_leds = LED_STRIP_LED_NUMBERS,        // The number of LEDs in the strip,
+        .strip_gpio_num = LED_STRIP_GPIO_PIN,   // The GPIO that connected to the LED strip's data line
+        .max_leds = LED_STRIP_LED_COUNT,        // The number of LEDs in the strip,
         .led_pixel_format = LED_PIXEL_FORMAT_GRB, // Pixel format of your LED strip
         .led_model = LED_MODEL_WS2812,            // LED strip model
         .flags.invert_out = false,                // whether to invert the output signal
+        .config_pixel_order = 0,                // The order of the pixel color. Not set or set to 0 if the default order is used
```

Comment:
can we showcase how to use the macro to set the pixel order, even though it's same as the default @Kainarx